src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -144,6 +148,7 @@ export class MdAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
 
     this._panelOpen = false;
     this._resetPlaceholder();
+    this._changeDetectorRef.detectChanges();
```

Comment:
I'll add a comment. It seems like using `fromEvent` doesn't trigger change detection, or at least it doesn't trigger it at the proper time. This means that the placeholder doesn't get reset when you click on the backdrop.